src/main/kotlin/com/openlattice/auditing/AuditRecordEntitySetsManager.kt

```diff
@@ -65,7 +66,7 @@ class AuditRecordEntitySetsManager(
             HazelcastMap.AUDIT_RECORD_ENTITY_SETS.name
     )
 
-    private val organizationTitles = hazelcastInstance.getMap<UUID, String>(
```

Comment:
I think the map name here still need to be changed to `ORGANIATIONS`

Reply:
*ORGANIZATIONS. if you change it to ORGANIATIONS then it will not compile. lol

Reply:
WHoops